examples/canvas_geometry_panorama.html

```diff
@@ -33,13 +33,14 @@
 		<div id="info"><a href="http://threejs.org" target="_blank">three.js</a> - panorama demo. cubemap by <a href="http://www.zfight.com/" target="_blank">Jochum Skoglund</a>.</div>
 
 		<script src="../build/three.js"></script>
+		<script src="js/libs/stats.min.js"></script>
```

Comment:
The order we have been following is like so. Flies included as-needed.
```
<script src="../build/three.js"></script>
<script src="js/controls/OrbitControls.js"></script>

<script src="js/loaders/OBJLoader.js"></script>

<script src="js/Detector.js"></script>
<script src="js/libs/stats.min.js"></script>
<script src='js/libs/dat.gui.min.js'></script>
```
I think it is best if we stick to consistent patterns in the examples -- for a whole host of reasons.